chips/cc26x2/src/rfc.rs

```diff
@@ -0,0 +1,580 @@
+//! RF Core
+//!
+//! Provides communication with the core module of the radio.
+//!
+//! The radio is managed by an external Cortex-M0 running prioprietary code in order to manage
+//! and set everything up. All stacks is implemented on this external MCU, and interaction
+//! with it enables the radio for communication in Sub-GHz bands.
+//!
+//! In order to communicate, we send commands to the Cortex-M0 through something called
+//! "Radio Doorbell".
+//!
+//!
+use commands as cmd;
+use core::cell::Cell;
+use kernel::common::regs::{ReadOnly, ReadWrite};
+use kernel::common::StaticRef;
+use prcm;
+
+//*****************************************************************************
+//
+// This section defines the register offsets of
+// RFC_DBELL component
+//
+//*****************************************************************************
+
+#[repr(C)]
+pub struct RfcDBellRegisters {
+    // Doorbell Command Register
+    pub cmdr: ReadWrite<u32>,
```

Comment:
Hrm... We should flag this as needing some serious attention. As it is, this low-level interface lends itself to being useful unsafely, and in fact, `post_cmdr` uses it in a way that can trivially break type safety in a number of ways.

Ideally this is looks something like:

```rust
pub struct RfcDBellRegisters {
    cmdr: VolatileCell<CmdrValue>
    ...
}
...

enum CmdrValue {
    Direct(u16),
    Command(&'static SomeDataStructure),
}
```

but that's no so trivial...

Comments below for at least how to mitigate the issues with exposing this unsafety outside the module for now.